src/graph/unit_graph.cc

```diff
@@ -629,10 +629,12 @@ class UnitGraph::CSR : public BaseHeteroGraph {
     return EdgeArray{row, coosubmat.col, coosubmat.data};
   }
 
-  EdgeArray Edges(dgl_type_t etype, const std::string &order = "") const override {
+  EdgeArray Edges(dgl_type_t etype, const std::string &order = "") override {
     CHECK(order.empty() || order == std::string("srcdst"))
       << "CSR only support Edges of order \"srcdst\","
       << " but got \"" << order << "\".";
+    if (!adj_.sorted)
+      aten::CSRSort_(&adj_);
```

Comment:
I don't like the idea of removing `const` qualifier because of this. One way to solve this is when creating CSR, always creating a sorted CSR. Let me ask around.